src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int BALL_NUMBER = 100;
+    private int ballNumber;
+
+    public Ball getRandomBall() {
+        ColorSupplier secondColor = new ColorSupplier();
```

Comment:
* ballNumber var can be just a local method var;
* would be better to reuse the same `ColorSupplier` and not to create a new one for each method call
* missing access modifiers
* naming
```suggestion
    private static final int BALL_MAX_NUMBER = 100;
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Ball getRandomBall() {
        ColorSupplier colorSupplier = new ColorSupplier();
```